locales/en/cms.ftl

```diff
@@ -1,194 +1,466 @@
+# Headline for Signin Unblock Code Page
+l10n-5-SigninUnblockCodePage-headline-68115606 = Enter authorization code
+# Primary Button Text for Signin Unblock Code Page
+l10n-5-SigninUnblockCodePage-primaryButtonText-7f83df9c = Authorize
+# Description for Signup Confirm Code Page
+l10n-5-SignupConfirmCodePage-description-8286e168 = Enter confirmation code for your Mozilla account
+# Headline for Signup Confirm Code Page
+l10n-5-SignupConfirmCodePage-headline-d076f12b = Enter confirmation code
+# Primary Button Text for Signup Confirm Code Page
+l10n-5-SignupConfirmCodePage-primaryButtonText-70d9be9b = Confirm
+# Headline for Signup Confirmed Sync Page
+l10n-5-SignupConfirmedSyncPage-headline-74db8403 = Sync confirmed
+# Primary Button Text for Signup Confirmed Sync Page
+l10n-5-SignupConfirmedSyncPage-primaryButtonText-598649e6 = Add another device
+# Headline for Signup Set Password Page
+l10n-5-SignupSetPasswordPage-headline-47e5e3c7 = Create a password to start a scan
+# Primary Button Text for Signup Set Password Page
+l10n-5-SignupSetPasswordPage-primaryButtonText-a0bfb8e5 = Continue
+# Description for Verify Login Code Email
+l10n-5-VerifyLoginCodeEmail-description-cdde5397 = Help us keep your account safe by approving your sign in on:
+# Headline for Verify Login Code Email
+l10n-5-VerifyLoginCodeEmail-headline-461a0bbf = Confirm your account to use Mozilla Monitor
+# Subject for Verify Login Code Email
+l10n-5-VerifyLoginCodeEmail-subject-461a0bbf = Confirm your account to use Mozilla Monitor
+# Description for Verify Short Code Email
+l10n-5-VerifyShortCodeEmail-description-2526309c = Use the code below to confirm your account.
+# Headline for Verify Short Code Email
+l10n-5-VerifyShortCodeEmail-headline-461a0bbf = Confirm your account to use Mozilla Monitor
+# Subject for Verify Short Code Email
+l10n-5-VerifyShortCodeEmail-subject-461a0bbf = Confirm your account to use Mozilla Monitor
+
+## l10n-6 - Firefox Desktop - Password Manager
+# Description for Email First Page
+l10n-6-EmailFirstPage-description-78fd76ac = Use your Mozilla account to securely sync to another device.
+# Headline for Email First Page
+l10n-6-EmailFirstPage-headline-0ec7d798 = Get saved passwords everywhere
+# Logo Alt Text for Email First Page
+l10n-6-EmailFirstPage-logoAltText-e9e92d2e = passwords sync
+# Primary Button Text for Email First Page
+l10n-6-EmailFirstPage-primaryButtonText-be11c74c = Get started
+# Headline for New Device Login Email
+l10n-6-NewDeviceLoginEmail-headline-7315250e = Your Mozilla account was used to sign in
+# Subject for New Device Login Email
+l10n-6-NewDeviceLoginEmail-subject-0a9e868b = New sign in to Firefox
+# Email From Name for Shared
+l10n-6-shared-emailFromName-763f7f1a = Firefox
+# Email Logo Alt Text for Shared
+l10n-6-shared-emailLogoAltText-96e4284e = Firefox logo
+# Page Title for Shared
+l10n-6-shared-pageTitle-5edcf763 = Set up Firefox Sync
```

Comment:
Sync should never be uppercase
https://acorn.firefox.com/latest/content/word-list/s-t-D8DB7Wjd